ibm_catalog.json

```diff
@@ -0,0 +1,98 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-observability",
+        "label": "Deployment of IBM Cloud Observability Resources",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
+          "solution"
+        ],
+        "keywords": [
+          "slz",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "observability",
+          "observability instances",
+          "log analysis",
+          "cloud monitoring"
+        ],
+        "short_description": "Deploys the IBM Cloud Observability Resources",
+        "long_description": "Deploys the IBM Cloud Observability Resources - Log Analysis and Cloud Monitoring",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-da/main/images/observability-icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Creates the Log analysis instance.",
+            "description": "Creates and configures the Log analysis instance."
+          },
+          {
+            "description": "Creates the Cloud Monitoring instance.",
+            "title": "Creates and configures the Cloud Monitoring instance."
+          },
+          {
+            "title": "Creates the COS bucket required for the logs archiving",
+            "description": "Creates and configures the COS bucket with archiving and expiration enabled."
+          },
+          {
+            "title": "Creates the COS bucket required for routing of ATracker events",
+            "description": "Creates and configures the COS bucket with archiving and expiration enabled."
+          }
```

Comment:
This are specific to the instances offering, so not sure they should be listed here too